styles.css

```diff
@@ -476,5 +488,240 @@ input:focus {
 .home-indicator {
   width: 134px;
   border-radius: 10px;
-  margin-top: 100px;
+  margin-top: 30px;
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  html {
+    box-sizing: border-box;
+  }
+
+  .container {
+    width: auto;
+  }
+
+  .nav-toggler {
+    display: none;
+  }
+
+  .head {
+    width: 100%;
+    background-color: #fff;
+    position: fixed;
+  }
+
+  .navbar {
+    height: 72px;
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  .menu-items {
+    display: flex;
+    justify-content: space-around;
+    width: 250px;
+  }
+
+  /* headline section */
+  .headline {
+    height: 100vh;
+    background-image: url(./src/assets/img/desktop-head-back.svg);
+    background-size: contain;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .head-wraper {
+    margin-top: -120px;
+  }
+
+  .head-primary {
+    width: 920px;
+    height: 122px;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .head-intro {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    align-items: center;
+  }
+
+  /* Projects section */
+  .projects {
+    padding: 142px;
+    gap: 142px;
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  .project-card {
+    width: auto;
+
+    /* height: 496px; */
+    flex-direction: row;
+    align-items: center;
+    box-sizing: border-box;
+    padding-bottom: 16px;
+  }
+
+  .snapshoot img {
+    width: 544px;
+    height: 448px;
+  }
```

Comment:
- [ ]  The cards are not displaying correctly here:

![image](https://user-images.githubusercontent.com/20567503/179373908-ac11a37d-2975-41af-aa7b-e6a6f622394d.png)

Kindly fix their styling so they are showing up correctly.